### PR TITLE
Feature/add location alert history

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Summary | Résumé

Location Outbreaks History view.

This is based largely on the Outbreaks History Search results display. Includes changes to the Place profile, Alert history, and Alert details screens.

To access, search for a Place, then on the Place profile, select "Show history"

Note: the "Back" link on the Exposure Details page currently takes you back to the History Search. There are now two entry points to this page, making the Back link a bit trickier to implement.

| Place profile | Alert history | Alert details |
| ----- | ----- | ----- |
image | image | image

### CODE REVIEW

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -914,33 +914,62 @@ msgstr "Trier par adresse, par ordre croissant"
...
+msgid "Exposure date"
+msgstr "Date de l’exposition"
+
+#: outbreaks/templates/location_history.html
+msgid "Sort by date alert sent, ascending"
+msgstr ""
+
+#: outbreaks/templates/location_history.html
+msgid "Sort by date alert sent, descending"
+msgstr ""
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:956]: @EliseKa svp

File: locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -914,33 +914,62 @@ msgstr "Trier par adresse, par ordre croissant"
...
+msgid "Alert sent"
+msgstr "Date d’envoi de l’alerte"
+
+#: outbreaks/templates/location_history.html
+msgid "Sort by sender, ascending"
+msgstr ""
+
+#: outbreaks/templates/location_history.html
+msgid "Sort by sender, descending"
+msgstr ""
```
Comment (dsamojlenko) [inline locale/fr/LC_MESSAGES/django.po:968]: @EliseKa svp

File: locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (EliseKa) [reply locale/fr/LC_MESSAGES/django.po:948]:
```suggestion
#: outbreaks/templates/location_history.html
msgid "Sort by date alert sent, ascending"
msgstr "Trier par date d'envoi de l'alerte, ordre croissant"

#: outbreaks/templates/location_history.html
msgid "Sort by date alert sent, descending"
msgstr "Trier par date d'envoi de l'alerte, ordre décroissant"
```

File: locale/fr/LC_MESSAGES/django.po (hunk 2 above)
Comment (EliseKa) [reply locale/fr/LC_MESSAGES/django.po:960]:
```suggestion
#: outbreaks/templates/location_history.html
msgid "Sort by sender, ascending"
msgstr "Trier par émetteur, ordre alphabétique"

#: outbreaks/templates/location_history.html
msgid "Sort by sender, descending"
msgstr "Trier par émetteur, ordre alphabétique inverse"
```

File: outbreaks/views.py [hunk 3]
```diff
@@ -592,6 +593,68 @@ def get_context_data(self, *args, **kwargs):
...
+            date_entry_tmpl = _("{} from {} to {}")
+            start_dmy_fmt = "%e %B %Y"
+            start_dmy = start_date.strftime(start_dmy_fmt)
+            start_hm = start_date.strftime(get_time_format(get_language()))
+            end_hm = end_date.strftime(get_time_format(get_language()))
+            notification_txt = date_entry_tmpl.format(start_dmy, start_hm, end_hm)
+            context["exposure_date_time"] = notification_txt
+            
+            # TODO: not crazy about this
+            self.request.session["alert_location"] = notification.location.id.hex
```
Comment (dsamojlenko) [inline outbreaks/views.py:611]: Feeling a bit weird about this ... in the existing functionality, we are setting a session variable on the Location Profile page, then the rest of the outbreak-creation steps use the session variable.

Now we have an alternative entry to outbreak-creation steps, from this Exposure Details view.

Really, I think we should probably refactor to include the Location ID on the URL for all of the outbreak creation steps. We should also refactor our URLs so they make sense. (ie, `outbreaks/[id]/profile` should be `location/[id]/profile`)

Comment (dsamojlenko) [reply outbreaks/views.py:610]: I think there are some improvements to URLs we could make, but for this particular concern, we should probably leave as-is. not broke/don't fix, etc...

**Review by dorianjp — APPROVED** [review]
lgtm

**Pull request merged**